frames/src/main/java/com/checkout/frames/cvvcomponent/CVVComponentFactory.kt

```diff
@@ -0,0 +1,25 @@
+package com.checkout.frames.cvvcomponent
+
+import android.content.Context
+import com.checkout.base.model.Environment
+import com.checkout.frames.cvvcomponent.api.CVVComponentApi
+import com.checkout.frames.cvvcomponent.api.CVVComponentApiClient
+
+public object CVVComponentFactory {
+
+    /**
+     * Creates [CVVComponentApi]
+     *
+     * @param publicKey - used for client-side authentication in the SDK
+     * @param context - represent the application context
+     * @param environment - [Environment] represent the environment for tokenization
+     */
+    @JvmStatic
+    public fun create(
+        publicKey: String,
+        environment: Environment,
+        context: Context,
+    ): CVVComponentApi {
+        return CVVComponentApiClient(publicKey, environment, context)
+    }
```

Comment:
Curious why don't we return the mediator here?

Reply:
As discussed in the refinement. Currently, we only load one CVV component for one card only. However, if merchants would like to load multiple CVV components for multiple cards they can do as an additional functionality. So, CVVComponentApiClient provides capabilities of loading multiple ComponentMediator for CVV tokenisation along with its individual CVV component.